### DISCUSSION TITLE
remove CyberChef nerd comments

### DISCUSSION THREAD

**Discussion by innovate-support** [Security-Onion-Solutions/securityonion#3054] [Ideas] [answered]
I was working through our system with a coworker today and he opened CyberChef.  During the loading spinner it showed an alarming message about Bitcoin Mining.  After some investigation it seems that it's just nerd code that's been inserted on purpose by the creators as a joke.  But this is not cool to have popup in a security tool's interface.  I suggest you strip it out of your CyberChef branch.

Here's the messages listed in the header code.  

**"Proving P = NP...",
"Computing 6 x 9...",
"Mining bitcoin...",
"Dividing by 0...",
"Initialising Skynet...",
"[REDACTED]",
"Downloading more RAM...",
"Ordering 1s and 0s...",
"Navigating neural network...",
"Importing machine learning...",
"Issuing Alice and Bob one-time pads...",
"Mining bitcoin cash...",
"Generating key material by trying to escape vim...",
"for i in range(additional): Pylon()",
"creating unresolved tension...",
"Symlinking emacs and vim to ed...",
"Training branch predictor...",
"Timing cache hits...",
"Speculatively executing recipes..."**

**Comment by dougburks** [comment, marked as answer]
We don't maintain our own CyberChef branch. Please feel free to make this suggestion directly to the CyberChef team:
https://github.com/gchq/CyberChef
